pdr_backend/pdr_dashboard/util/format.py

```diff
@@ -16,6 +16,8 @@
     {"name": "Source", "id": "source"},
     {"name": "Timeframe", "id": "timeframe"},
     {"name": "Full Addr", "id": "contract"},
+    {"name": "Avg Accuracy", "id": "avg_accuracy"},
```

Comment:
Let's rename this to 'Accuracy' to have it consistent with the predictoors table. Maybe we should also change it into the Feeds page table as well

Reply:
is it the same formatting?